### PR TITLE
fix: duplicate toasts

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/Notifications/PeerNotifications.tsx [hunk 1]
```diff
@@ -20,7 +20,8 @@ export const PeerNotifications = () => {
   const [selectedPeer, setPeerSelector] = useSetSubscribedChatSelector(CHAT_SELECTOR.PEER);
 
   useEffect(() => {
-    if (!notification?.data) {
+    // @ts-ignore
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Notifications/PeerNotifications.tsx:23]: why ts-ignore is needed? also we can combine both the conditions

File: packages/roomkit-react/src/Prebuilt/components/Toast/ToastBatcher.js [hunk 2]
```diff
@@ -40,7 +41,13 @@ export const ToastBatcher = {
   },
   showToast({ notification, duration = 3000, type }) {
     try {
-      this.showToastInternal({ notification, duration, type });
+      if (!this.toastCache[notification.id]) {
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Toast/ToastBatcher.js:44]: i am not sure this covers all cases

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/Toast/ToastBatcher.js:44]: Should we merge to QA to DF?

Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/Toast/ToastBatcher.js:44]: we can df on vercel itself.

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**